src/wikmd/wikmd-config.yaml

```diff
@@ -12,7 +12,7 @@ main_branch_name: "main"
 sync_with_remote: 0
 remote_url: ""
 
-wiki_directory: "wiki"
+wiki_directory: "../wiki"
```

Comment:
I'm not sure if this is clear. Config file is inside `src/wikmd/wikmd-config.yaml` But we really have a wiki folder in `../../wiki`. I get that this has to do with the python modules structure, but I think that might be confusing.

Reply:
That is very true. We should probably keep this as simple as possible. However that means that we would need to define the default development locations as environment variables.

This we could easily do with the `python-dotenv` package and a file within the root of the project create a file called `.env` where we define these variables.

Reply:
It's worth mentioning that if we can say that people should use the `--editable` way of working as I mentioned below then we could make away with the `../wiki` and have it as `wiki` as at point you are running it from the root (meaning that you can't even use the `../wiki` path as that owuld point outside your project).

Reply:
Indeed, thats why I suggested running the module outside of src folder. I think that might be the best solution.